_communityposts/test_post.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
I suppose this is good for documentation (or as a **template**) but it should not be included in production. Maybe we can have a folder, like `templates`, extract this file there instead? So that when making a post, we know what structure is expected.

Reply:
We can just hide communityposts.md in the navigation_pages list until we have a post that we want to publish

Reply:
Done!